src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -578,19 +560,13 @@
   <data name="Profile_ScrollbarVisibilityVisible.Content" xml:space="preserve">
     <value>Visible</value>
   </data>
-  <data name="Profile_SelectionBackgroundColor.Header" xml:space="preserve">
-    <value>Selection background color</value>
-  </data>
-  <data name="Profile_SelectionBackgroundColorToolTip.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Sets the background color of selected text. Overrides the selection background set in the color scheme.</value>
-  </data>

```

Comment:
Oh, were these three just not used at all (hence removing them?)

Reply:
Yeah, I wanted to ask about that actually. We weren't using them (because we cut a few settings) so I removed them. Found them while I was updating the resw file. But, we do plan on adding them back in at some point. I feel like the right move is to remove them but just want to double check.

Reply:
Since we have to break the loc keys, let's remove them. We _will_ probably want to bring them back eventually... but we can do it when we get there.